app/src/main/java/rick_and_morty/ui/characters/CharactersActivity.kt

```diff
@@ -0,0 +1,19 @@
+package rick_and_morty.ui.characters
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.example.rick_and_morty.R
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class CharactersActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_character)
```

Comment:
nit: instead of this, we can pass the layout resource in the `AppCompatActivity` constructor

Reply:
done